flocker/restapi/_infrastructure.py

```diff
@@ -25,6 +23,10 @@
 from ._logging import LOG_SYSTEM, REQUEST, JSON_REQUEST
 from ._schema import getValidator
 
+__all__ = [
+    "EndpointResponse", "structured", "user_documentation",
+    ]
+
```

Comment:
:question: This is a  private module. Does it need an `__all__` ...at all? We do this in the package ``__init__.py`` file right?